### PR TITLE
Add instructions to install and run GameController

### PR DESCRIPTION
**Pull request by JosephusPaye**
[Preview](https://deploy-preview-112--nubook.netlify.app/guides/tools/webots-setup#install-game-controller)

### CODE REVIEW

**Review by wongjoel — DISMISSED** [review]
LGTM

**Comment by LachlanCourt** [comment]
Do we want to add the kill java process thing in here?

File: src/book/03-guides/02-tools/03-webots.mdx [hunk 1]
```diff
@@ -104,6 +104,42 @@ sudo apt-get install cmake-curses-gui libprotobuf-dev protobuf-compiler libeigen
...
+
+   ```sh
+   sudo apt-get install ant default-jdk
+   ```
+
+2. Clone the Robocup TC GameController repo:
+
+   ```sh
+   cd ~
+   git clone https://github.com/RoboCup-Humanoid-TC/GameController
```
Comment (KipHamiltons) [inline src/book/03-guides/02-tools/03-webots.mdx:133]:
```suggestion
   git clone https://github.com/RoboCup-Humanoid-TC/GameController.git
```

**Comment by KipHamiltons** [comment]
We should merge this soon @JosephusPaye

File: src/book/03-guides/02-tools/03-webots.mdx [hunk 2]
```diff
@@ -173,6 +209,38 @@ sudo apt-get install cmake-curses-gui libprotobuf-dev protobuf-compiler libeigen
 
    2. In the pop-up, navigate to the [NUbots/NUWebots](https://github.com/NUbots/NUWebots/) repository on your computer and open the `worlds/kid.wbt` world file.
 
+## Running Webots with GameController
+
+1. Find your primary local IP address by running the following.
+
+   ```sh
+   hostname -I
```
Comment (LachlanCourt) [inline src/book/03-guides/02-tools/03-webots.mdx:217]: This didn't work on arch I don't think... @KipHamiltons What did you end up using instead?

Comment (KipHamiltons) [reply src/book/03-guides/02-tools/03-webots.mdx:217]: I think I used `ip addr`. I don't think we should bother adding that to this guide though. Arch users can fend for themselves in this case ;p

**Review by KipHamiltons — APPROVED** [review]
I made a branch which built our fork of webots on mac (`mac-build`). It'd be worth adding that or probably making a new PR mentioning it actually.

**Pull request merged**